x-pack/plugins/canvas/public/state/actions/elements.js

```diff
@@ -104,23 +103,18 @@ export const fetchContext = createThunk(
     });
 
     const variables = getWorkpadVariablesAsObject(getState());
-
+    const { expressions } = pluginServices.getServices();
```

Comment:
referencing services in actions seems like a bad idea...but at this point it's not any worse than what was there previously. We should think about how to get rid of that moving forward.